eclipse-che/che-theia
# fix(webpack): webpack 5 build

Signed-off-by: Thomas Mäder <tmader@redhat.com>



### What does this PR do?

* Updates the webpack build of the front-end che API to Webpack 5
* Add the necessary dependencies for building with Webpack 5 in a non-source linked case

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/20083

### How to test this PR?
See the linked issue  for setup instructions. Build che-theia and run che-theia according to instructions.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=next
